plugins/lighthouse/src/components/CategoryTrendline/index.test.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable jest/no-disabled-tests */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import { wrapInThemedTestApp } from '@backstage/test-utils';
+
+import CategoryTrendline from '.';
+
+describe('CategoryTrendline', () => {
+  describe('when no data is present', () => {
+    it('renders null without throwing', () => {
+      const rendered = render(
+        wrapInThemedTestApp(<CategoryTrendline data={[]} title="sparkline" />),
+      );
+      expect(rendered.queryByTitle('sparkline')).not.toBeInTheDocument();
+    });
+  });
+
+  describe('when one datapoint is present', () => {
+    it('renders as a straight line', () => {
+      const rendered = render(
+        wrapInThemedTestApp(
+          <CategoryTrendline data={[0.5]} title="sparkline" />,
+        ),
+      );
+      expect(rendered.getByTitle('sparkline')).toBeInTheDocument();
+    });
+  });
+
+  describe.skip('when the data finishes above the success threshold', () => {
```

Comment:
full disclosure: not really sure how to test this, so it is skipped. maybe I fish around and use expect(HTMLElement).toHaveAttributes(), but it'd be better as a viz regression test with storybook or something.